src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/components/root/main.js

```diff
@@ -177,7 +195,7 @@ define(['sulumedia/model/media'], function(Media) {
                 },
                 {
                     el: this.$find(constants.datagridSelector),
-                    url: '/admin/api/media?orderBy=media.changed&orderSort=DESC',
+                    url: '/admin/api/media?orderBy=media.changed&orderSort=DESC&locale=' + this.locale,
```

Comment:
the language chooser in the "root" is also new! this was one of the two issues which i fix with this PR. i will change the DESC to desc!